cmd/buildtools/openebs.go

```diff
@@ -78,3 +78,115 @@ var updateOpenEBSAddonCommand = &cli.Command{
 		return nil
 	},
 }
+
+type addonComponent struct {
+	name                        string
+	wolfiPackageName            string
```

Comment:
nit: how about just always using `getWolfiPackageName` and changing its function header to something like:
```golang
func(args ...string) string
```